README.md

```diff
@@ -1,35 +1,72 @@
-[![Apache License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
-# Salesforce ([docs](https://dbt-salesforce.netlify.app/)) 
 
-This package models Salesforce data from [Fivetran's connector](https://fivetran.com/docs/applications/salesforce). It uses data in the format described by [this ERD](https://docs.google.com/presentation/d/1fB6aCiX_C1lieJf55TbS2v1yv9sp-AHNNAh2x7jnJ48/edit#slide=id.g3cb9b617d1_0_237).
-
-The main focus of this package is enable users to better understand the performance of your opportunities. You can easily understand what is going on in your sales funnel and dig into how the members of your sales team are performing. 
-
-## Models
-
-The primary outputs of this package are described below. Staging and intermediate models are used to create these output models. This package contains transformation models, designed to work simultaneously with staging models in our [Salesforce source package](https://github.com/fivetran/dbt_salesforce_source).
-
-**model**|**description**
+<p align="center">
+    <a alt="License"
+        href="https://github.com/fivetran/dbt_salesforce/blob/main/LICENSE">
+        <img src="https://img.shields.io/badge/License-Apache%202.0-blue.svg" /></a>
+    <a alt="Fivetran-Release"
+        href="https://fivetran.com/docs/getting-started/core-concepts#releasephases">
+        <img src="https://img.shields.io/badge/dbt_core-version_>=1.0.0_<2.0.0-orange.svg" /></a>
+    <a alt="Maintained?">
+        <img src="https://img.shields.io/badge/Maintained%3F-yes-green.svg" /></a>
+    <a alt="PRs">
+        <img src="https://img.shields.io/badge/Contributions-welcome-blueviolet" /></a>
+</p>
+
+# Salesforce Modeling dbt Package ([Docs](https://fivetran.github.io/dbt_salesforce/))
+
+# 📣 What does this dbt package do?
+- Produces modeled tables that leverage Salesforce data from [Fivetran's connector](https://fivetran.com/docs/applications/salesforce) in the format described by [this ERD](https://fivetran.com/docs/applications/salesforce#schema) and builds off the output of our [Salesforce source package](https://github.com/fivetran/dbt_salesforce_source).
+
+- This package enables users to:
+  - Understand the performance of your opportunities
+  - Understand what is going on in your sales funnel and 
+  - Drill into how the members of your sales team are performing
+  - Have a daily view of sales activities 
+  - Access an enhanced contact list
```

Comment:
"Access" sounds a bit strange in the context of the models we are providing. Can we change this to "leverage" instead?